feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -314,26 +303,31 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
     return this.createTicketCommand.execute(ticketStarted);
   }
 
-  getTicketsState(
-    pageSize: number,
-    currentPage: number,
-    sort: string
-  ): Observable<QueryState<TicketList | undefined>> {
-    return this.getTicketsQuery$(pageSize, currentPage, sort).getState();
-  }
-
   getTickets(
     pageSize: number,
     currentPage: number,
     sort: string
   ): Observable<TicketList | undefined> {
-    return this.getTicketsState(pageSize, currentPage, sort).pipe(
-      map((state) => state.data)
+    return this.customerTicketingListPreConditions().pipe(
+      switchMap((customerId) =>
+        this.customerTicketingConnector.getTickets(
+          customerId,
+          pageSize,
+          currentPage,
+          sort
+        )
+      )
     );
   }
 
-  createTicketEvent(ticketEvent: TicketEvent): Observable<TicketEvent> {
-    return this.createTicketEventCommand.execute(ticketEvent);
+  createTicketEvent(
+    ticketEvent: TicketEvent,
+    containsAttachment?: boolean | false
```

Comment:
this is a pretty weird typing. I'm assuming you want to have a default value set if no 2nd param is in right?

Then the following is how you should do it 
```suggestion
    containsAttachment?: boolean = false
```

Reply:
It does not accept `=` as a intializer

Reply:
```suggestion
    containsAttachment = false
```

Reply:
only reason why it didn't work is because the `?` optional was there. Only needed to remove `?` and should work. Can't have both a default value and it being optional because it's indirectly optional with the default value